### ISSUE TITLE
Improve the disposition summary on MURs

### ISSUE THREAD

**Issue by adrianwebb** [18F/fec-eregs#294]
Comment: 

Summary and Disposition Information.  Understandably the disposition information is not uniform, particularly between MURs in EQS and archived MURs given the different sources this information was pulled from.  In general, the Disposition information that appears in the Summary for many of the non-MUR archived cases is not particularly helpful.  In seems to consist of all certifications of Commission votes.   For example, in MUR 6597 (Durkee), the disposition in the Summary section includes the final certification where the Commission accepted a conciliation agreement and the first certification where it found reason to believe she violated the Act.  The first certification information is unnecessary.  In contrast the Disposition section under that is helpful:

Disposition: Conciliation-PPC
Respondent: Durkee, Kinde

If this information comes from Law Manager that may be a better source. I note though this information is not always present.  See for example the Disposition section for MUR 5903 (PBSJ) which is blank. 

For more comments see: https://docs.google.com/document/d/1eVFGVEDJEOESyjOptWO58uYJkg7NB26xF6rK9fCQE6E/edit

**Comment by adrianwebb** [comment]
Amy to schedule meeting to clarify some of the comments about how this could be more helpful.

**Comment by nickykrause** [comment]
(cc: @jenniferthibault, @emileighoutlaw, @adrianwebb)
   
The design team just connected about legal issues, in an attempt to remove any blocks that may exist from us this sprint, so I'm updating this issue with a summary of what we discussed:

It seems fairly clear that the user is concerned about the utility of the information that appears in the disposition section of the "Summary." (As opposed to the dedicated "Disposition" area beneath it.)

As a part of the user's feedback, they suggested another place where more relevant information could be found: Law Manager. 

Consequently, our current thinking is that it does not yet seem necessary to speak with the user(s) to gather more clarity on their concerns. Instead, it might be helpful to do the following:

- [ ] Assess whether or not the summary information that we're providing is taken from the best source possible
- [ ] Assess the feasibility of finding a new source for the summaries (possibly Law Manager)

^ These two tasks will likely require some conversation with the FEC, because, as we discussed during the initial review of the MURs feedback, it seems like there have been concerns historically about using information from Law Manager.

If no source of useful information can be found, then we could do some design thinking:

- [ ] If the summary disposition is repeatedly surfacing information that is not useful, or that is hard to read, and we can't access law Manager or get useful information from somewhere, then maybe we should consider eliminating the summary disposition altogether.

**Comment by jenniferthibault** [comment]
I've unearthed a few more clues for discussing possible data sources for the Disposition in the Summary section with the FEC.

Currently, I _believe_ the Disposition in the Summary section is taken from the "Commission action" field in the Enforcement Query System (EQS) database:

Compare: https://beta.fec.gov/data/legal/matter-under-review/6261/

<img width="511" alt="screen shot 2016-11-02 at 12 30 42 am" src="https://cloud.githubusercontent.com/assets/11636908/19916759/a0ee066c-a093-11e6-88bf-debc0972d5ec.png">


to:

<img width="721" alt="screen shot 2016-11-02 at 12 26 56 am" src="https://cloud.githubusercontent.com/assets/11636908/19916697/1b51dd30-a093-11e6-8fad-087157c2b475.png">


The data for the design as it was initially intended was to come from the FEC Record. View the history of that moving to an enhancement in [issue #71](https://github.com/18F/fec-eregs/issues/71#issue-151429122) I'm not certain about _why_ this wasn't brought into the first implementation, or if there are blockers. We may need to tap @edmullen to find out.


So while we know this issue still isn't a requirement for the first release, the improvement we can make is to learn from the FEC if the FEC Record, the Law Manager, or some other place is a better place to retrieve _useful_ summary information, and if we have clearance (from OGC or whoever) to use that source instad

**Comment by jenniferthibault** [comment]
Amy is starting to open conversations about this. Moving to backlog until ready.

**Comment by edmullen** [comment]
Hmm. Issue #71 was related to the AOs, and you're talking about MURs here. Piecing together the original thinking...

The original hope was not to draw the top summary from the FEC Record or the Commission Action, but from the **Weekly Digest** which publishes plain language text that could be used for that whole top summary area. [Look here](http://www.fec.gov/press/press2016/digests/201601028digest.shtml) in the Enforcement area.

We got some initial positive feedback that the top Summary area content was helpful, as based in the [prototype](https://gsa.invisionapp.com/share/AR7U3Y662#/167711271_MUR_Canonical) on the Weekly Digest content,  See https://github.com/18F/fec-eregs/issues/122#issuecomment-226612208 in MUR Case Canonical page.

I think implementation happened right after I rolled off. I assume it was blocked by the challange of using the Weekly digest content. Porta would know more.

**Comment by jenniferthibault** [comment]
Thanks @edmullen — you are correct about the AO/MUR mixup. I was able to chat with Porta and straightened myself out. Amy is opening conversations about the **Weekly Digest**

**Comment by nickykrause** [comment]
@jenniferthibault: I was just going back over these issues and was curious if there is an update on this one. When we were onsite, did Amy happen to mention anything about pulling this Weekly Digest content into MURs?

(We could also just wait to discuss this in grooming today...thought I'd ask here, though, in case you see this and have time between now and then)

**Comment by jenniferthibault** [comment]
I spoke with Amy on Nov 2nd about this over Slack—
- The FEC does have MUR summaries from the press office's weekly digests that cover every MUR--even ones from the archive. This is commission-approved language, which is a plus. It's all kept in a Word document right now, which is a minus.
- the beta site has scraped Weekly Digests for all MURs from the 2000's, but not before. So we'd still need to get this data from everything before 2000

We left off at this assessment, and Amy was going to think about how to best approach the FEC's office of General Counsel about this. I don't believe it's moved, because there have been other priorities, and I'd want to wait to nudge it until this task comes closer to being assigned a sprint again. Since it's an enhancement to the current design, I'm not sure how that fits in with current PI planned work...my sense is that it may not be. Let's check this in today's grooming?

**Comment by nickykrause** [comment]
Update on this issue: Until we can decide on a better alternative source for the `Disposition` information within the MUR `Summary`, we will be removing that information from the interface. 

See https://github.com/18F/fec-eregs/issues/325

**Comment by nickykrause** [comment]
Another update here: We found out from the FEC that turning off this information may cause problems for users. So, we have been investigating other ways to display it (possibly as a `Votes` section, rather than as a part of the `Summary`). We are exploring the canonical page adjustments over in https://github.com/18F/fec-eregs/issues/326. 

@noahmanger Unless you disagree, I think we could close this issue in favor of #326?

**Comment by nickykrause** [comment]
Wait @noahmanger - scratch that. I think this should be a different issue that addresses pulling the Weekly Digest summaries for MURs, _as well as_ the FEC Record summaries for AOs. These seem like related ideas and both of them are known issues. I will update in a moment...

**Comment by nickykrause** [comment]
Closing in favor of https://github.com/18F/fec-eregs/issues/327
